content/docs/intro/concepts/config.md

```diff
@@ -433,63 +433,20 @@ $ pulumi stack init my-stack \
-
-```
-$ pulumi config --show-secrets --json > config.json
-$ pulumi stack export --show-secrets > stack.json
-```
-
-Next, you will need to create a temporary new stack using the desired new secrets provider.  If using the `passphrase` secrets provider, this can also be used to create a new passphrase to use for encryption.
-
-```
-$ pulumi stack init temp --secrets-provider <provider>
-```
-
-If you chose `passphrase` you will see something like this:
-
-```
-$ cat Pulumi.temp.yaml
-encryptionsalt: v1:h8kauiCOXZc=:v1:0Tjy4uHvwgw/b0Hp:gKAqRJ74vyIkc5BeEUlCFXynwRTPFQ==
-```
-
-Else if you chose one of the cloud-provider secrets providers you will see something like this:
-
-```
-$ cat Pulumi.temp.yaml
-secretsprovider: awskms://alias/my-kms-key
-encryptedkey: AQICAHhivZtfYgNwNJDVWDd++88uo2qgy5MJqqzsPfTFv7uzDgF+1kaw9/wSKgGh5QpT/cwtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMc2EiH7Mli5ohzB1aAgEQgDsOszkuuC7wffN3FdtKCNAcRoJq3krWmeA/ucnSMn/F2OYVIDKxDZkjJYZBAjEmpWL49Fcly/n8NMNHhw==
+```bash
+$ pulumi stack change-secrets-provider "<secrets-provider>"
 ```
 
-And if you chose the Pulumi Service secrets provider, there will be no stack file.
-
-You will use the values above to replace those in use in your existing stack.  Switch back to the existing stack with `pulumi stack select ...`, then update your config and state as outlined below.
-
-First, your config.  
-
-1. You will need to remove all encypted secrets from your current config using `pulumi config rm` or by removing them by hand from the `.yaml` file (you saved them to `config.json` above).  
-2. Then copy the lines above (`encryptionsalt` or `secretsprovider`+`encryptedkey`) from the `temp` stack into your primary stack (replacing the existing corresponding lines).
-3. Save that, and then run `pulumi config set --secret <key> <value>` for each of the secret config values you removed, using the values from `config.json`.  
-
-Next, your state.  
-
-1. Open `stack.json` and find the `secrets_providers` section.  
-2. Based on the output of your `Pulumi.temp.yaml` pick from these options:
-    * For `encryptionsalt` set `"type": "passphrase",` and replace the contents of the `state` with `"salt": "<value>"`.
-    * For `secretsprovider`+`encryptedkey` set `"type": "cloud"` and replace the contents of the `state` with `"url": "<value>"` and `"encryptedkey": "<value>"` respectively.
-    * For Pulumi Service secrets-provider, set `"type": "service"` and replace the contents of the `state` with `"url"`, `"owner"`, `"project"` and `"stack"` as appropriate for your existing stack.
+This will change the encrypted secrets in the provider configuration and the stack's state file. The supported secrets providers are:
 
-Save your `stack.json` with those changes, then run `pulumi stack import < stack.json` to import it into your Pulumi stack.
+* default (Pulumi KMS)
```

Comment:
```suggestion
* `default`
```

Also - probably should make all of these quoted since they are the formal names?

I was also going to suggest linking these to somewhere where we defined how to use each one - but couldn't find great docs on that.  We may want to either link to https://www.pulumi.com/docs/reference/cli/pulumi_stack_init/ or copy some of the content there to a topic in the main docs and then reference that from here?

Reply:
I added a link to the docs in the stack init page